projects/packages/forms/src/contact-form/class-contact-form-endpoint.php

```diff
@@ -1028,7 +1028,7 @@ public function get_forms_config( WP_REST_Request $request ) { // phpcs:ignore V
 			'siteURL'                 => ( new Status() )->get_site_suffix(),
 			'hasFeedback'             => ( new Forms_Dashboard() )->has_feedback(),
 			'hasAI'                   => $has_ai,
-			'enableIntegrationsTab'   => apply_filters( 'jetpack_forms_enable_integrations_tab', true ),
```

Comment:
We were using this filter on wpcom to hide integrations. We've updated this to use the new filter already in 191699-ghe-Automattic/wpcom ...